pkg/broker/transport/producer.go

```diff
@@ -5,12 +5,19 @@ import (
 	"fmt"
 	"log/slog"
 
-	"github.com/GenesisEducationKyiv/software-engineering-school-4-0-Hukyl/currency-rate/internal/mail/transport/config"
+	"github.com/GenesisEducationKyiv/software-engineering-school-4-0-Hukyl/pkg/broker/transport/config"
 	amqp "github.com/rabbitmq/amqp091-go"
 )
 
+const (
+	durable    = false
+	autoDelete = false
+	exclusive  = false
+	noWait     = false
+)
+
 func logAndWrap(msg string, err error) error {
```

Comment:
does this function make sense? In most cases (I suppose, but not sure) after returning the error, you're logging it somewhere at a higher level. This function returns an error and logs it on the lower level, with the possibility that this error could also be logger on the higher level. Kind of duplication of the logs